pkgs/development/tools/cmake-format/default.nix

```diff
@@ -10,12 +10,15 @@
 
 buildPythonApplication rec {
   pname = "cmake-format";
-  version = "0.6.11";
+  version = "0.6.13";
+  # The source distribution does not build because of missing files.
```

Comment:
I think we should package `cmakelang` and add an alias to `cmake-format`

Reply:
I think we should wait and see if that name actually sticks and the author follows through with that change.